### ISSUE TITLE
Cannot detect the title, exec or class of the window on Ubuntu 22.04.2 Wayland with Sway

### ISSUE THREAD

**Issue by BjornW** [espanso/espanso#1552]
**Describe the bug**
Detect displays a new window but the detected values are empty. 

**To Reproduce**
1. type "#detect#" (without quotes) in an application such as Firefox search bar or terminal or LibreOffice
2. A window pops open with the title, exec and class values empty 

**Expected behavior**
I expect to find the detected window title, exec and class to be shown. 

**Screenshots**
![screenshot-2023-03-17_18-37-03](https://user-images.githubusercontent.com/65042/225978254-413f5d59-1d8a-41e1-bc23-f25f67afbff8.png)

**Setup information**
- OS:  Ubuntu 22.04.2 LTS with Wayland and Sway as window manager
- Version: 2.1.8

**Comment by BjornW** [comment]
In addition: 
I'd like to use an application specific configuration to make LibreOffice behave and use the Clipboard backend while other applications can keep using the default backend. Seems LibreOffice will only work correctly with the clipboard back. Without the option to filter an application on title or class this will be not possible afaik.

**Comment by bkuri** [comment]
Same issue using Arch+Hyprland.

BTW the command `rofi -show window` works great to detect window titles, so perhaps something similar could be used with espanso? Just a thought.

**Comment by zchap26** [comment]
Experiencing the exact same behaviour on Debian 11 with Wayland and Gnome:

1. cannot detect title, exec or class, making app-specific configurations impossible

2. espanso will only work correctly in LibreOffice when using the clipboard backend

Are these perhaps two unrelated bugs, or is LibreOffice behaviour expected?

Regards

**Comment by putzwasser** [comment]
Can confirm for Manjaro Linux v23.1.0 - kernel: 5.15.138-1-MANJARO and Wayland.

**Comment by federico-terzi** [comment]
Yep, unfortunately the `info` module is not implemented on Wayland yet: https://github.com/espanso/espanso/blob/dev/espanso-info/src/wayland/mod.rs

**Comment by kohane27** [comment]
Thank you for the update! I'll keep an eye when this gets update. Thank you again for this amazing program!
